src/main/java/com/coveros/selenified/selenium/WaitFor.java

```diff
@@ -719,7 +719,7 @@ public int elementNotDisplayed(Element element, double seconds) {
             // wait for up to XX seconds
             double end = System.currentTimeMillis() + (seconds * 1000);
             while (System.currentTimeMillis() < end) {
-                if (!is.elementPresent(element, false) || !webElement.isDisplayed()) {
```

Comment:
this can be accomplished. think of this usecase:
Element is displayed on the page, element is hidden (second usage that you left)
Element is displayed on the page, element is removed (first usage that you removed)